src/main/java/seedu/revision/ui/RestoreWindow.java

```diff
@@ -0,0 +1,118 @@
+package seedu.revision.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.input.Clipboard;
+import javafx.scene.input.ClipboardContent;
+import javafx.stage.Stage;
+import seedu.revision.commons.core.LogsCenter;
+
+import java.net.URL;
+import java.util.logging.Logger;
+
+/**
+ * Controller for a restore page.
+ */
+public class RestoreWindow extends UiPart<Stage>{
```

Comment:
Instead of creating a new window for Restore. Is it possible to use an Alert object like in StartQuizWindow#handleEnd? I think there's much lesser code and a bit cleaner.